docs/history/hatch.md

```diff
@@ -34,7 +34,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ***Added:***
 
 - Add ability to control the source of Python distributions
-- Upgrade Ruff to 0.6.8
+- Upgrade Ruff to 0.4.5
```

Comment:
Er, to clarify I meant that the "Upgrade Ruff to 0.6.8" just shouldn't be under the `1.12.0` heading, but rather `Unreleased`.

Reply:
Yes thank you for clarifying, I did understand but couldn't easily change that within the UI so I choose to remove it